decidim-core/app/views/pages/home/_statistics.html.erb

```diff
@@ -16,6 +16,30 @@
               <span class="home-pam__number processes-count"><%= participatory_processes.count %></span>
             </div>
           </div>
+          <div class="home-pam__highlight">
```

Comment:
It comes from `decidim-design`:

https://github.com/decidim/design/blob/1fd6a731d1e7f7d5ea3ddda487412be259efbf9a/source/home.html.erb#L248

Although +1 on this, we should probably change this name and update it on the design repo.